emgapi/migrations/0010_run_assembly_split.py

```diff
@@ -88,29 +88,30 @@ def populate_assemblies(apps, schema_editor):
         if AnalysisJob.objects.filter(run=run).count() < 1:
             run.delete()
 
-# def delete_duplicated(apps, schema_editor):
-#     Run = apps.get_model("emgapi", "Run")
-#     ExperimentType = apps.get_model("emgapi", "ExperimentType")
-#     experiment_type = ExperimentType.objects.get(experiment_type="assembly")
-#
-#     for run in Run.objects.filter(experiment_type=experiment_type):
-#         run.delete()
 
+def add_experiment_types(apps, schema_editor):
+    experiment_types = ['metatranscriptomic', 'metagenomic', 'amplicon', 'assembly', 'metabarcoding', 'unknown']
+    Experiment_type = apps.get_model('emgapi', 'ExperimentType')
+    for i, t in enumerate(experiment_types):
+        e = Experiment_type(experiment_type_id=i + 1, experiment_type=t)
+        e.save()
 
-class Migration(migrations.Migration):
 
+class Migration(migrations.Migration):
     dependencies = [
         ('emgapi', '0009_remove_gsccvcv'),
     ]
 
     operations = [
+        migrations.RunPython(add_experiment_types),
         migrations.CreateModel(
             name='Assembly',
             fields=[
                 ('assembly_id', models.BigAutoField(db_column='ASSEMBLY_ID', primary_key=True, serialize=False)),
                 ('accession', models.CharField(blank=True, db_column='ACCESSION', max_length=80, null=True)),
                 ('wgs_accession', models.CharField(blank=True, db_column='WGS_ACCESSION', max_length=100, null=True)),
-                ('legacy_accession', models.CharField(blank=True, db_column='LEGACY_ACCESSION', max_length=100, null=True)),
+                ('legacy_accession',
```

Comment:
could you please revert all the formatting changes? THis is autogenerated code and I would prefer if remain original. Migrations are not flake8-ed